homeassistant/components/dwd_weather_warnings/sensor.py

```diff
@@ -60,46 +55,52 @@
         icon="mdi:close-octagon-outline",
     ),
 )
-MONITORED_CONDITIONS: list[str] = [desc.key for desc in SENSOR_TYPES]
-
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
You can't remove the schema here or it will not load from YAML

Reply:
I removed this, because the variable was not used anywhere in this file. The YAML configuration gets imported automatically, which worked in my tests. Is this really necessary?

Reply:
It's used when loading config.yaml and without it, it will never start `async_setup_platform` as there would not be any loaded YAML.

Reply:
Then I'm confused 😄 I just debugged it and the function was called for every 5 of my YAML sensors. All of them were imported successfully without having the schema. I have 5 sensors in the `configuration.yaml` and `async_setup_platform` as well as `async_setup_entry` were called 5 times

Reply:
That seems very weird how it would load without a schema.
Anyway it should be there for validation even if it somehow works without it.